authentication/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class CustomUserAdmin(UserAdmin):
+    pass  # 空的管理类，以便注册自定义的用户模型
+
+
+admin.site.unregister(User)
```

Comment:
i got this error when running server
```
  File "/Users/yunchipang/dev/cs5200/goodhome/venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 153, in unregister
    raise NotRegistered("The model %s is not registered" % model.__name__)
django.contrib.admin.exceptions.NotRegistered: The model User is not registered
```
not sure what the issue is, but should be something related to this line here. could you look into it?

Reply:
user model import conflicts, now fixed